### PR TITLE
Update Maven wrapper to 3.5.0

### PR DESCRIPTION
**Pull request by ArturGajowy**
Release notes: https://maven.apache.org/docs/3.5.0/release-notes.html

### CODE REVIEW

File: mvnw [hunk 1]
```diff
@@ -219,16 +209,28 @@ concat_lines() {
...
+fi
+
+# Provide a "standardized" way to retrieve the CLI args that will
 # work with both Windows and non-Windows executions.
 MAVEN_CMD_LINE_ARGS="$MAVEN_CONFIG $@"
 export MAVEN_CMD_LINE_ARGS
 
 WRAPPER_LAUNCHER=org.apache.maven.wrapper.MavenWrapperMain
 
+# avoid using MAVEN_CMD_LINE_ARGS below since that would loose parameter escaping in $@
```
Comment (findepi) [inline mvnw]: Is it your comment or upstream's? ( `s/oo/o/` )

Comment (ArturGajowy) [inline mvnw]: upstream, generated by mvn wrapper plugin

**Comment by ArturGajowy** [comment]
@electrum please have a look :)

File: .mvn/wrapper/maven-wrapper.properties [hunk 2]
```diff
@@ -1 +1 @@
-distributionUrl=https://repo1.maven.org/maven2/org/apache/maven/apache-maven/3.3.9/apache-maven-3.3.9-bin.zip
+distributionUrl=https://repo1.maven.org/maven2/org/apache/maven/apache-maven/3.5.0/apache-maven-3.5.0-bin.zip
```
Comment (electrum) [inline .mvn/wrapper/maven-wrapper.properties]: Let's add a newline at the end

**Comment by electrum** [comment]
Merged, thanks!

**Pull request closed without merging**